### ISSUE TITLE
Debug mixed_layer_depth POD: module 'xarray' has no attribute 'ufuncs'

### ISSUE THREAD

**Issue by Wen-hao-Dong** [NOAA-GFDL/MDTF-diagnostics#419]
**Bug Severity**  
* [x ] **1 = Minor problem** that does not affect total framework functionality (e.g., computation error in a POD, problem with logging output, or an issue on a single system  
* [ ] **2 = Major problem** that affects overall functionality, but that does not occur for all users (e.g., problems installing the framework with a specific Conda version, a framework option that causes one or more PODs to fail, or missing/incompatible Python modules).  
* [ ] **3 = Catastrophic problem** that occurs frequently for multiple users and/or on multiple systems (e.g.,framework consistently fails to install on multiple systems, or one or more PODs continuously fails after running successfully) 

**Describe the bug**
When testing this POD with ESM4 output, File "/arch0/wnd/MDTF-GFDL/mdtf/MDTF-diagnostics/diagnostics/mixed_layer_depth/mixed_layer_depth.py", line 229, in compute mld
 mld = mld.where(~xr.ufuncs.isnan(mld), bottom_depth)
 AttributeError: module 'xarray' has no attribute 'ufuncs'

**Steps To Reproduce**
I am using the updated version and have submitted the job using the following command: sbatch ./mdtf_slurm -f sites/NOAA_GFDL/default_gfdl.jsonc --site NOAA_GFDL -v

**Environment**
Describe the system environment:
- OS: [e.g. RHEL 7.2]
- Conda version: [the `conda info` command will provide the installation information] 
- branch name and link: [git hash/branch: b29208a8a108eb1dfa4e8bc039e2d6eef34981ae (on test0716)]

**Log information and/or terminal output**
/archive/wnd/MDTF-GFDL/wkdir/MDTF_ESM4_historical_D1_1980_2014.v3/mixed_layer_depth/mixed_layer_depth.log

**Comment by wrongkindofdoctor** [comment]
@Wen-hao-Dong I fixed this issue in a recent update. Sync your branch(es) with the latest main branch and this should fix the error.

**Comment by Wen-hao-Dong** [comment]
@wrongkindofdoctor You are right, the above issue has been resolved after updating my branch. But now it failed due to TimeoutError: [Errno 110] Connection timed out
Please see the log file here:/archive/wnd/MDTF-GFDL/wkdir/MDTF_ESM4_historical_D1_2000_2004.v6/mixed_layer_depth/mixed_layer_depth.log
